packages/@ama-sdk/schematics/README.md

```diff
@@ -79,12 +79,45 @@ If you use `Yarn2+`, you can use the following `scripts` in `package.json`:
 
 Use `generate` to (re)generate your SDK based on the content of `./swagger-spec.yaml` (make sure you have this file at the root of your project) and `upgrade:repository` to regenerate the structure of your project.
 
+#### Light SDK
+
+It is also possible to generate a "light" SDK. This type of SDK does not need to generate revivers, which results in a simplified version of the code and reduces the size of the bundle.
+
+Please note that revivers are generated for SDKs that use:
+* non-stringified dates
+* dictionaries
+* extensible models
+
+##### Dates
+If your specification file includes dates, there are multiple options for the generation of your SDK involving the global property option `stringifyDate`: 
+* By default, the option `stringifyDate` is false so dates will be generated as either `utils.Date`, `utils.DateTime`, or `Date`. 
```

Comment:
How is done the difference when generating a javascript `Date` or an `utils.Date`. Is it another option to be passed to the generator ?

Reply:
I believe this is explained in the doc that I linked in the line below, the difference depends on the x-date-timezone vendor in the specs (https://github.com/AmadeusITGroup/otter/tree/main/packages/%40ama-sdk/schematics/schematics/typescript/shell/templates/base#manage-dates)

Reply:
ok I see :) I missed that info.